webapp/channels/src/packages/mattermost-redux/src/utils/timezone_utils.ts

```diff
@@ -21,9 +21,9 @@ export function getUserCurrentTimezone(userTimezone?: UserTimezone): string {
     }
 
     if (useAutomatic) {
-        return automaticTimezone;
+        return automaticTimezone || 'UTC';
```

Comment:
Why are these fallbacks needed? Based on the type definition, this should always be defined

Reply:
I think it was failing on tests, and to be sure we are not breaking anything I added that fallback into the code.